web/app/view/GroupDialog.js

```diff
@@ -40,5 +40,18 @@ Ext.define('Traccar.view.GroupDialog', {
             displayField: 'name',
             valueField: 'id'
         }]
-    }
+    },
+
+    buttons: [{
```

Comment:
We can have custom bottom bar for geofences, but share same implementation for other dialogs. If it doesn't work, I guess you missed something. I don't see why it wouldn't work.